pkg/distro/bootc/bootc.go

```diff
@@ -140,6 +140,12 @@ func (d *BootcDistro) addArches(arches ...*BootcArch) {
 	}
 }
 
+func NewBootcArch(arch arch.Arch) *BootcArch {
+	return &BootcArch{
+		arch: arch,
+	}
+}
+
```

Comment:
This function does not seem to be used anywhere and it is added by a seemingly unrelated commit.